ios/Sources/AboutFeature/AboutDroidKaigiScreen.swift

```diff
@@ -0,0 +1,87 @@
+import SwiftUI
+import Styleguide
+
+public struct AboutDroidKaigiScreen: View {
+    @Environment(\.presentationMode) var presentationMode
+
+    public init() {
+        UITableView.appearance().backgroundColor = .clear
+        UINavigationBar.appearance().backgroundColor = .clear
+        UINavigationBar.appearance().barTintColor = .clear
+    }
+
+    public var body: some View {
+        NavigationView {
+            VStack {
+                VStack {
+                    Text(L10n.AboutDroidKaigiScreen.whatIs).font(.system(size: 22))
+                    AssetImage.logoTitle.image
+                        .frame(width: 261.81, height: 41.06, alignment: .center)
+                    Text(L10n.AboutDroidKaigiScreen.description)
+                        .multilineTextAlignment(.center)
+                        .font(.system(size: 17))
+                        .opacity(0.7)
+                }
+                .padding(.horizontal, 32)
+
+                List {
+                    Button(action: {}, label: {
+                        HStack() {
+                            Text(L10n.AboutDroidKaigiScreen.behaviorCode)
+                            Spacer()
+                            Image(systemName: "book")
+                        }
+                    })
+                    .listRowBackground(AssetColor.Background.contents.color)
+                    .foregroundColor(AssetColor.Base.secondary.color)
+
+                    Button(action: {}, label: {
+                        HStack() {
+                            Text(L10n.AboutDroidKaigiScreen.opensourceLincense)
+                            Spacer()
+                            Image(systemName: "star")
+                        }
+                    })
+                    .listRowBackground(AssetColor.Background.contents.color)
+                    .foregroundColor(AssetColor.Base.secondary.color)
+
+                    Button(action: {}, label: {
+                        HStack() {
+                            Text(L10n.AboutDroidKaigiScreen.privacyPolicy)
+                            Spacer()
+                            Image(systemName: "magnifyingglass")
+                        }
+                    })
+                    .listRowBackground(AssetColor.Background.contents.color)
+                    .foregroundColor(AssetColor.Base.secondary.color)
+                }
+                .listStyle(InsetGroupedListStyle())
+
+            }
+            .background(AssetColor.Background.secondary.color.ignoresSafeArea())
```

Comment:
Please fix like this 🙏 
https://github.com/DroidKaigi/conference-app-2021/pull/498/files#diff-d5d1cb73fe87e2cd47f97568ed5dde318895b17f4118aed9d501cd28626b5135R58-R60